src/md/datasource/datatargetreader.cpp

```diff
@@ -163,13 +163,10 @@ void DataTargetReader::Align(DWORD alignmentBytes)
 
 void DataTargetReader::AlignBase()
 {
-#ifdef _MSC_VER
-    // Windows MSVC compiler aligns structs based on the largest field size
+    // Align structs based on the largest field size
+    // This is the default for MSVC compilers
+    // This is forced on other platforms by the DAC_ALIGNAS macro
```

Comment:
Is the macro is already in place for every structure we care about? The comment suggested we were adding it now but maybe I am misunderstanding the intent.

Reply:
The `DAC_ALIGNAS` is already in most places that need it. The alignment it enforced on Windows vs Linux + the forced alignment on the `datatargetreader` was what cause the regressions. Steve had added the alignment changes in https://github.com/dotnet/coreclr/pull/28035/files#diff-1e0b6e2c3dafc15c663fd7ba67a7e529R193 as part of the crossdac work after his tool reported layout misalignments.

Reply:
The intent of #28035 was to fix this layout discrepancy.  So they all should be fixed.

Reply:
👍